mqtt/mqtt3/README.md

```diff
@@ -20,6 +20,37 @@ The crate is not published to crates.io yet. Please generate docs locally with `
 
 See the `examples/` directory for examples of a publisher and subscriber, and for how to set a will.
 
+# Fuzz testing
+
+The crate is coming with fuzz tests. 
+
+## Prerequisites
+
+```bash
+sudo apt install tmux binutils
+cargo install afl
```

Comment:
This needs to match the `afl` dep of the crate, so make this `cargo install afl --version '^0.6'`. Then it won't break when afl 0.7 is released.